runtimes/eden/Cargo.toml

```diff
@@ -40,7 +40,7 @@ std = [
   "pallet-contracts-primitives/std",
   "pallet-contracts-rpc-runtime-api/std",
   "pallet-contracts/std",
-  #"pallet-xcm/std",
+  "pallet-xcm/std",
```

Comment:
This pallet should be listed under "try-runtime" with `pallet-xcm/try-runtime` as well.

Reply:
Done